components/contribution-flow/ContributionBlocker.js

```diff
@@ -49,6 +51,16 @@ const msg = defineMessages({
 /**
  * From received params, see if there's anything preventing the contribution
  */
+export const Redirect = ({ to }) => {
```

Comment:
I think this `to` needs to be added to `Proptypes`. The linter complains about it as well. 😃